cmd/vclusterctl/cmd/create.go

```diff
@@ -232,9 +216,41 @@ func (cmd *CreateCmd) Run(ctx context.Context, args []string) error {
 		newExtraValues = append(newExtraValues, tempValuesFile)
 	}
 
+	// Check if the passed in values adhere to our config format.
+	for _, p := range newExtraValues {
+		f, err := os.Open(p)
+		if err != nil {
+			return err
+		}
+		defer f.Close()
+
+		cfg := &config.Config{}
+		err = cfg.Validate(f)
+		if err != nil {
+			cmd.log.Infof("Consider using %q to convert the old values format to the new 0.20 format", "vcluster migrate values")
```

Comment:
This might not always be the case, as we also return errors when unable to ReadAll or convert the YAML to JSON.

We might want to introduce a new, concrete error `var ErrInvalidFileFormat = errors.New("invalid file format")` and wrap this error and the decode error together, then check here using `errors.Is` if the actual error returned.